variables.tf

```diff
@@ -82,3 +82,8 @@ variable "tags" {
   description = "Tags that will be assigned to all resources."
   default     = {}
 }
+
+variable "additional_regions" {
+  type    = set(string)
+  default = []
```

Comment:
A short description of what this does and why it's sometimes necessary would be good.

Reply:
added description